src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
```

Comment:
The bold font variant is missing in the `@font-face` declaration. Please ensure that the bold font variant is included and used as specified in the task requirements.